install_arduino_teensyduino.sh

```diff
@@ -0,0 +1,54 @@
+#/usr/bin/env bash
+
+# NOTE: Running this script in sudo will cause the checks for environment vars to fail
+# To complete the installation run `sudo install.sh` in the arduino IDE folder
+
+ARDUINO_IDE_VERSION="1.8.12"
+TEENSYDUINO_VERSION="152"
+ARDUINO_IDE_FOLDER="$HOME/arduino-$ARDUINO_IDE_VERSION"
+ARDUINO_DOWNLOAD_URL="https://downloads.arduino.cc/arduino-$ARDUINO_IDE_VERSION-linux64.tar.xz"
+TEENSYDUINO_DOWNLOAD_URL="https://www.pjrc.com/teensy/td_$TEENSYDUINO_VERSION/TeensyduinoInstall.linux64"
+
+count=`ls -1 ./arduino-$ARDUINO_IDE_VERSION-* 2>/dev/null | wc -l`   
```

Comment:
You have two `count` variables, perhaps one should be like count_of_leftover_files or changing the names to distinguish things out a bit more